app/scripts/head/startup-styles.js

```diff
@@ -0,0 +1,137 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Adds styles on app startup. Used to eliminate startup screen flicker
+// by adding classes to the HTML element before any content is rendered.
+// Run from the HEAD element of the page.
+
+// NOTE: This file must run in IE8+, so no ES5/ES6 features!
+
+//jshint es3: true, es5: false, esnext: false
+
+'use strict';
+
+// This is loaded in the HEAD of the doc & uses a modified version of
+// https://github.com/umdjs/umd/blob/master/amdWeb.js
+(function (root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    // AMD. Register as an anonymous module.
+    define(['lib/environment'], factory);
+  } else {
+    // Browser globals
+    root.FxaHead.StartupStyles = factory(root.FxaHead.Environment);
+  }
+}(this, function (Environment) { //jshint ignore: line
+  function parseQueryParams(queryParams) {
+    var search = queryParams.replace(/^\?/, '');
+    var paramPairs = search.split('&');
+    var params = {};
+
+    // Use old school for instead of Array.prototype.forEach because
+    // env-test still has to run in IE8 even if the rest of the
```

Comment:
should `env-test` be renamed now? Another mention of it here: 
![image](https://cloud.githubusercontent.com/assets/128755/7278879/26be5272-e8e6-11e4-9f9a-906ee5eec5be.png)